testsuite/features/step_definitions/api_common.rb

```diff
@@ -0,0 +1,622 @@
+# Copyright (c) 2015-2022 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+require 'json'
+require 'socket'
+
+$api_test = APITestHTTP.new($server.full_hostname)
+
+## auth namespace
+
+When(/^I am logged in API as user "([^"]*)" and password "([^"]*)"$/) do |user, password|
+  $api_test.auth_login(user, password)
+end
+
+When(/^I logout from API$/) do
+  $api_test.auth_logout
+end
```

Comment:
We don't really need, unless we specially want to cover the test of this.
Let's keep a unique connection open during the whole test suite.

Reply:
Again, see concern above.